src/bundle/Resources/config/services/controllers.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+    _defaults:
+        autoconfigure: true
+        autowire: true
+        public: false
+
+    Ibexa\Bundle\Search\Controller\SearchController:
```

Comment:
we often use:
```
    Ibexa\Bundle\PackageName\Controller\:
        resource: './../../Controller'
        tags:
            - controller.service_arguments
```

Reply:
I've observed a different approach in this package.

Reply:
Perhaps because it had only one controller defined in `services.yaml` file. As you move the definition to a dedicated file and introduce a second controller you could change it. If those controllers are not 'specific' I see no point in defining them one by one